src/main/java/com/emc/pravega/perf/PravegaTransactionWriterWorker.java

```diff
@@ -34,30 +34,33 @@
     PravegaTransactionWriterWorker(int sensorId, int eventsPerSec,
```

Comment:
I think this class could just extend WriterWorker directly.

Reply:
no, here the PravegaTransactionWriterWorker need the producer created by PravegaWriterWorker.

Reply:
Performing transaction on a Producer is a deprecated API. The new API works by creating a transactional producer from the client factory directly.

Reply:
yes, but i will do as part of  : https://github.com/pravega/pravega-benchmark/issues/16